tests/clients/test_pep0249.py

```diff
@@ -0,0 +1,320 @@
+import logging
+from typing import Generator, TYPE_CHECKING
+from unittest.mock import patch
+import psycopg2
+import psycopg2.extras
+import pytest
+from instana.instrumentation.pep0249 import (
+    ConnectionFactory,
+    ConnectionWrapper,
+    CursorWrapper,
+)
+from opentelemetry.trace import SpanKind
+from instana.singletons import tracer
+from instana.util.traceutils import get_tracer_tuple
+from pytest import LogCaptureFixture
+from instana.span.span import InstanaSpan
+
+
+class TestCursorWrapper:
+    @pytest.fixture(autouse=True)
+    def _setup(self) -> Generator[None, None, None]:
```

Comment:
Can we rename this since it does both setup and teardown?

```suggestion
    def _resource(self) -> Generator[None, None, None]:
```

Reply:
Sure, changing.